### ISSUE TITLE
Support for SOCOM: U.S. Navy SEALs Confrontation

### ISSUE THREAD

**Issue by DeadOnAim** [isJuhn/KAMI#88]
Was wondering if it would be possible to get support for BCUS98152, BCUS98183, and BCES00173

https://www.psone.online/getting-started

Any help would be greatly appreciated it would be awesome to get good mouse and keyboard control for this game, right now you cannot aim properly or shoot with the default rpcs3 settings.

**Comment by DeadOnAim** [comment]
To be honest, if KAMI works well with PCSX2 maybe add support for SOCOM I, SOCOM II, SOCOM III, and Combined Assault Im sure members of those communities would be happy if they had the option to use their mouse as well.

**Comment by Eagle1One** [comment]
I agree. That would be greatly appreciated. I know a few players who cannot use a controller cause of an injury, they would love it!
